src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1277,7 +1277,13 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 }
             }
 
-            // Add the content headers
+            // If this is a Get request and there is no content, then don't fill in the content as empty content gets rejected by some web services
+            if (request.Method == HttpMethod.Get && request.Content == null)
```

Comment:
You could put this check for HttpMethod.Get and early out in the if statement at line 1287, and save the extra null check.
```csharp
if (request.Content is null)
{
    if (request.Method == HttpMethod.Get)
    {
        // If this is a Get request and there is no content, then don't fill in the content as empty content gets rejected by some web services
        return;
    }
    ...
```

Reply:
makes sense